__tests__/test.js

```diff
@@ -66,3 +69,27 @@ describe("test DB for requests", () => {
       });
   });
 });
+
+describe("GET /api/articles ", () => {
+  test("/api/articles returns all article in order of date", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((res) => {
+        expect(res.status).toBe(200);
```

Comment:
Please assert your status codes using the .expect(200) syntax. It's way neater and built into supertest so why not use it.